plugins/modules/postgresql_info.py

```diff
@@ -761,14 +761,20 @@ def get_ext_info(self):
         ext_dict = {}
         for i in res:
             ext_ver_raw = i[1]
-            ext_ver = i[1].split('.')
+            
+            if re.search('^([0-9]+([\-]*[0-9]+)?\.)+[0-9]+([\-]*[0-9]+)?$', i[1]) == None:
+                ext_ver = [None]
```

Comment:
```suggestion
                ext_ver = [None, None]
```
maybe? to avoid failing on lines 778

Anyway, if someone can help with manual testing, it'd be great

Reply:
ah, yes, sorry, thanks